src/data/markdown/docs/03 cloud/03 Manage/04 Environment variables.md

```diff
@@ -0,0 +1,95 @@
+---
+title: 'Environment variables'
+excerpt: 'How to use environment variables in the cloud'
+---
+
+Environment Variables are key-value pairs configured at the organization level. All values are encrypted before being stored in our database and remain encrypted until they are needed for test-runs. You reference environment variables within a test-script and the variables are interpolated on the server that runs the Test-Run. Environment Variables can be used to store sensitive (and non-sensitive) information that you want to reference in your test-scripts. Managing environment variables in one central place reduces the data exposure and ensures that the changes take effect in all test-runs that use these variables.
+
+> Changes to an Environment Variable won't apply to already started Test-Runs, they only apply to new upcoming Test-Runs.
+
+## Managing environment variables
+
+Environment Variables can be managed in Organization Settings: Select your profile icon, then **Manage > Environment variables**. Permission to create, reveal, modify and delete environment variables is restricted to Owners and Admins of the organization. 
+
+![k6 Environment Variable](./images/envvars/environment-variables.png)
+
+#### Naming environment variables
+
+The following rules must apply for naming an environment variable:
+- Names can only contain alphanumeric characters (`[a-z]`, `[A-Z]`, `[0-9]`) or underscores (`_`). Spaces are not allowed.
+- Names must not start with the `K6_CLOUD` prefix.
+- Names must not start with a number.
+- Names are not case-sensitive.
+- Names must be unique.
+
+#### Declaring a new environment variable
+
+Enter the Name for your Environment Variable. Then, enter the respective Value. Optionally, you can also add a brief description then click **submit**. 
+
+![k6 Environment Variable](./images/envvars/create-environment-variable.png)
+
+### Accessing environment variables
+
+The cloud Environment Variable can be referenced in the [script-editor](https://k6.io/docs/cloud/creating-and-running-a-test/script-editor/) through the global `__ENV` variable, the same way as in [k6 CLI Environment Variable](https://k6.io/docs/using-k6/environment-variables/#passing-environment-variables-to-the-k6-script)
+
+```javascript
+import http from 'k6/http';
+import { sleep } from 'k6';
+
+export default function () {
+  const params = {
+    headers: { Authorization: __ENV.AUTH_TOKEN },
+  };
+  const res = http.get(__ENV.SERVICE_URL, params);
+  sleep(1);
+}
+```
+
+<Blockquote mod="warning">
+It's essential to stress that the current environment variables implementation doesn't prevent printing values to the log (or restrict exposing a variable any other way). You should avoid printing sensitive data to the log intentionally.
+</Blockquote>
+
+## Running tests with k6 cloud command
```

Comment:
`Running tests with k6 cloud command` this section seems to be out of context.

Reply:
You mean out of context because what you mentioned below about `differences between the "Execution modes"` or the title does not make sense?

Reply:
@ppcano Why do you say so? I like the current positioning, since it seems like the logical workflow.

1. Find the page to manage variables.
2. Name one.
3. Reference it.
4. Run it.

Reply:
Apologies for being short on the review.

Yes, this order is logical:
- Manage (create) environment variables
- Reference it on the k6 script
- Run a cloud test


IMO, the logical order would be `Run a cloud test`: either using the Web App or CLI. Not `Run tests with k6 cloud command`.

This is why I find this section a little out of context.

1 - It describes only the case for running a cloud test with the CLI. 
2 -  I am biased but I find the following obvious:

> Environment variables set up on the cloud can be referenced on test runs run with k6 cloud command.

This is the only sentence in this section and don't add any additional information to me. Is this not what we said in the previous section?

> If you create an Environment Variable on k6 Cloud, you can reference it in your k6 script through the __ENV variable,


I'd consider either removing it or detailing more of the options to run a cloud test. But the latter could be redundant, so I lean more toward the former. 

Another alternative:

- Manage environment variables
- Access environment variables
- Order of preferences
- Limitations

Reply:
Oh. I didn't realize how short that section was. You're right.

> I'd consider ... removing 

Agreed.